spec/index.bs

```diff
@@ -6984,6 +6994,7 @@ enum GPUStoreOp {
                 1. Add |buffer| to the [=usage scope=] as [=internal usage/input=].
                 1. Set |this|.{{GPURenderEncoderBase/[[index_buffer]]}} to be |buffer|.
                 1. Set |this|.{{GPURenderEncoderBase/[[index_format]]}} to be |indexFormat|.
+                1. Set |this|.{{GPURenderEncoderBase/[[index_buffer_size]]}} to be |size|.
```

Comment:
`size` (and `offset`) are optional; we need to implement the defaulting behavior for both this and existing text to make sense.

Reply:
In the arguments description it states that if `size == 0`, `|buffer|.[[size]] - |offset|` is used, and we already use it as such in the validation block. I can move that down into the algorithm if you'd like.

Reply:
Oh, I didn't realize it was in the argument block. I think we should move it, and in the argument block say something like "defaulting to the full remaining size of the buffer". Also, I thought we had gotten rid of all of the sentinel zero values, but apparently we missed this one, so we should follow up to fix that.

Reply:
Moved. I'll leave the sentinel `0` for a different PR.